packages/next/client/image.tsx

```diff
@@ -166,18 +166,19 @@ export default function Image({
 
 type LoaderProps = {
   root: string
-  filename: string
+  src: string
   width?: number
 }
 
-function imgixLoader({ root, filename, width }: LoaderProps): string {
-  return `${root}${filename}${width ? '?w=' + width : ''}`
+function imgixLoader({ root, src, width }: LoaderProps): string {
+  return `${root}${src}${width ? '?w=' + width : ''}`
```

Comment:
Doesn't imgix have a quality parameter too?

Reply:
Yeah but it'll just default if not used.